packages/unit-testing/stubs/ui/serverWidget/Form.js

```diff
@@ -0,0 +1,403 @@
+define(['./Tab', './Button', './Field', './FieldGroup', './Sublist'], function (Tab, Button, Field, FieldGroup, Sublist) {
+    /**
+     * @class Form
+     * @classdesc Primary object used to encapsulate a NetSuite-looking form.
+     * After you create a Form object, you can add a variety of scriptable elements to the form including fields, links, buttons, tabs, and sublists.
+     * @protected
+     * @constructor
+     *
+     * @since 2015.2
+     */
+    function Form() {
+
+        /**
+         * The form title
+         * @name Form#title
+         * @type {string}
+         *
+         * @since 2015.2
+         */
+        this.title = undefined;
+        /**
+         * This method is called during a beforeLoad UE or a suitelet and the message is later displayed on the client side,
+         * once the pageInit script is completed. The method takes either an already created Message object or the options
+         * object that would be used for creating the message.
+         * User event context can be used to control whether the message is shown on records in view, create, or edit mode (not applicable for Suitelets).
+         * @restriction Server SuiteScript only
+         * @governance none
+         * @param {Object|Message} options config object OR the message options object to be displayed in browser, described in N/ui/message: create()
+         * @param {Message} options.message the message object to be displayed in browser, described in N/ui/message: create()
```

Comment:
This Message is refers to ui/message/MessageInstance, we could inject it.

Reply:
Yes